test/journal_test.go

```diff
@@ -89,6 +89,92 @@ func TestJournalCrSimple(t *testing.T) {
 	)
 }
 
+// bob creates a conflicting file while running the journal, and then
+// hits another conflict while trying to put the resolution.
+// Regression test for KBFS-1590.
+func TestJournalCrConflictOnPut(t *testing.T) {
```

Comment:
Hmm...I think since you wrote it already we could just keep it. If it starts causing problems, then we can remove it later.